app/models/finance/statement/npq.rb

```diff
@@ -6,6 +6,13 @@ class Finance::Statement::NPQ < Finance::Statement
   def payable!
     update!(type: "Finance::Statement::NPQ::Payable")
   end
+
+  def previous_statements
+    Finanace::Statement::NPQ
```

Comment:
```suggestion
    cpd_lead_provider.where(cohort:).where("payment_date < ?", payment_date)
```

Could these be moved up the main `Finance::Statement` class?

Reply:
Oh I see that there was already that method there. 
Correct me if I'm wrong, but it would have picked up the both the ECF and NPQ statements correct?